src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/repository/VideoAppointmentRepository.kt

```diff
@@ -24,7 +24,18 @@ interface VideoAppointmentRepository : ReadOnlyRepository<VideoAppointment, Long
     forLocationKeys: List<String>,
   ): List<VideoAppointment>
 
-  fun findByPrisonerNumberAndAppointmentDateAndPrisonLocKeyAndStartTimeAndEndTime(
```

Comment:
I could have just added the status code as part of the search but I really don't like these long function names albeit Spring generates the query for you.